activities/ingest_queue_populate.py

```diff
@@ -79,13 +86,20 @@ def ingest_populate(args):
                      poll_delay = POLL_DELAY,
                      status_delay = STATUS_DELAY)
 
-    total_sent = reduce(lambda x, y: x+y, results, 0)
+    tile_count = get_tile_count(args)
+    messages_uploaded = reduce(lambda x, y: x+y, results, 0)
+    if tile_count != messages_uploaded:
+        log.debug("Warning.  Messages uploaded do not match tile count.  tile count: {} messages uploaded: {}"
```

Comment:
Why not use `log.warning()` if the message is a warning?

Reply:
Updated.